services/github_service.py

```diff
@@ -1,3 +1,5 @@
+# pylint: disable=E1136, E1135, W0718, C0411
```

Comment:
Why are you disabling all of these?

Reply:
2 of em beginning with Es were already disabled. Am disabling C0411 because it was making spurious complaints about third party import order again, and W0718 because it was complaining about non-specific exceptions thrown, which have nothing to do with this PR and were there beforehand.